src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/controller/SupplierAssessmentController.kt

```diff
@@ -39,4 +41,21 @@ class SupplierAssessmentController(
       )
     )
   }
+
+  @GetMapping("sent-referral/{id}/supplier-assessment-appointment")
```

Comment:
same comment on the naming as before. 

if the supplier assessment id is on the referral dto then could we have `GET /supplier-assessment/{id}`